src/components/flatList.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { ListRenderItem } from 'react-native';
+import { SafeAreaView } from 'react-native';
+import { StyleSheet, View, FlatList, Text, StatusBar } from 'react-native';
+import { Journey } from '../model/fareResponseClass';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    marginTop: StatusBar.currentHeight || 0,
+  },
+  item: {
+    backgroundColor: '#f9c2ff',
+    padding: 10,
+    marginVertical: 2,
+    marginHorizontal: 4,
+  },
+  title: {
+    fontSize: 14,
+  },
+});
+
+type FlatListProps = {
+  items: Journey[] | null;
+};
+
+const FlatListComponent: React.FC<FlatListProps> = ({ items }) => {
+  const renderItem: ListRenderItem<Journey> = ({ item }) => {
```

Comment:
Given you have no logic in here other than an immediate return statement, you can shorthand this will the following syntax:

```typescript
const renderItem: ListRenderItem<Journey> = ({ item }) => (
  <View style={styles.item}>
    <Text style={styles.title}>{item.originStation.displayName}</Text>
    <Text style={styles.title}>{item.destinationStation.displayName}</Text>
    <Text style={styles.title}>{item.departureTime}</Text>
    <Text style={styles.title}>{item.arrivalTime}</Text>
  </View>
);
```